crates/pallets/starknet/src/lib.rs

```diff
@@ -928,7 +931,9 @@ impl<T: Config> Pallet<T> {
         let pending = Self::pending();
 
         let global_state_root = Felt252Wrapper::ZERO;
-        let sequencer_address = Self::sequencer_address();
+        // Maybe we can change sequencer_address to be Felt252Wrapper itself.
+        let sequencer_address =
```

Comment:
we should definitely yes, please create a `good first issue`

Reply:
no it should not, we're using the sequencer address in each transaction and converting it from Felt252Wrapper to bytes is expensive. If you want to change it you can change it to starkfelt but i don't think it adds any value and it probably implies a refacto on the blockifier side

Reply:
The starknet-rs `FieldElement` type is internally using the Montgomery representation which is expensive to convert to regular bytes

Reply:
feel free to revert the PR then, or lmk if its not possible to revert it directly

Reply:
feel free to revert the PR then, or lmk if its not possible to revert it directly i will open a PR